pep/package.json

```diff
@@ -43,7 +43,7 @@
         "@gavant/ember-floating-labels": "^1.1.0",
         "@gavant/ember-modals": "^1.0.3",
         "@gavant/ember-pagination": "3.0.0-beta.2",
-        "@gavant/ember-shopify-draggable": "^0.3.3",
+        "@gavant/ember-shopify-draggable": "https://github.com/Gavant/ember-shopify-draggable.git#update-addon",
```

Comment:
do you really need master to use this unpublished branch for the addon? if you think its ready to go can we just publish a beta release, or if you are trying to test stage/prod specific issues, using at least a specific commit hash here instead might be best. 

My worry is if you happen to push to that branch later on, subsequent prod builds will pick up those changes, so its a little volatile.

Reply:
Whoops,  that actually shouldn't be there. I can fix that